### DISCUSSION TITLE
the update of supervision

### DISCUSSION THREAD

**Discussion by OliverLam7725** [roboflow/supervision#1192] [Q&A]

**Comment by SkalskiP** [comment]
Hi @OliverLam7725 👋🏻 I don't understand. Could you be a bit more specific? :) What do you mean by `if you updated the items in supervision folder, please update the code as well`?

**Comment by OliverLam7725** [comment]
my bad, I misunderstand it. Please delete this form thx!

**Comment by SkalskiP** [comment]
@OliverLam7725 no worries ;)
